src/index.js

```diff
@@ -128,13 +129,22 @@ class Service {
 
 function raw (service, method, params) {
   // handle client methods like indices.create
-  const [meth, ext] = method.split('.');
-
-  if (typeof ext !== 'undefined') {
-    return service.Model[meth][ext](params);
+  const [primaryMethod, secondaryMethod] = method.split('.');
+
```

Comment:
I change the name as `primaryMethod` and `secondaryMethod`, hopefully it is fortunate.